packages/plugins/integration-activepieces/src/lib/activepieces-webhook.service.ts

```
@@ -0,0 +1,199 @@
+    try {
+      const subscription = await this.activepiecesWebhookSubscriptionRepository.findOne({
+        where: { id }
+      });
+
+      if (!subscription) {
+        return false;
+      }
+
+      // Simple ping to validate webhook endpoint
+      const response = await firstValueFrom(
+        this._httpService.get(subscription.targetUrl + '/health').pipe(
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)